iroh/src/rpc_protocol.rs

```diff
@@ -13,8 +21,15 @@ use serde::{Deserialize, Serialize};
 
 pub use iroh_bytes::provider::{ProvideProgress, ValidateProgress};
 
+/// A request to the node to provide the data at the given path
+///
+/// Will produce a stream of [`ProvideProgress`] messages.
 #[derive(Debug, Serialize, Deserialize)]
 pub struct ProvideRequest {
+    /// The path to the data to provide.
+    ///
+    /// This should be an absolute path.
+    /// The node and the cli should be on the same filesystem.
```

Comment:
nitpicking, but does the cli access this path?  is it not sufficient that this path exists on the node?